src/Hamiltonians/reduced_density_matrix.jl

```diff
@@ -344,3 +363,365 @@ function Base.iterate(od::TestOneParticleDensityOffdiagonals, state=(1, 1))
     end
     return nothing
 end
+
+"""
+     GradOneParticleDensity(v; normalize=true) <: AbstractOperator{SVector{m,T}}
+A one particle operator constructed from a provided test vector `v[1,:]` and 
+its gradient `v[2:m+1,:]` with respect to parameters `α₁,α₂,...,αₘ`. 
+An expectation value with this operator yields a gradient of the largest 
+eigenvalue of the one-particle density matrix. `T` is the eltype of `v`.
+If `normalize` is true (default), the vector is normalized before use.
+
+```math
+    \\partial_α ρ̂ {(1)}= ∑_{ij} (v_{i}^* \\partial_{\\bar(α)} v_{j} + 
+        \\partial_{\\bar(α)} v_{i}^* v_{j}) (â^†_{i} â_{j} - ζ δ_{i,j})
```

Comment:
Please use unicode where possible to improve readability in the editor/CL! E.g ∂_α ᾱ.